rasa/server.py

```diff
@@ -137,6 +137,25 @@ def decorated(*args, **kwargs):
     return decorator
 
 
+def ensure_conversation_exists(app: Sanic):
+    """Wraps a request handler ensuring the conversation exists.
+    """
+
+    def decorator(f):
+        @wraps(f)
+        def decorated(*args, **kwargs):
+            conversation_id = kwargs["conversation_id"]
+            tracker = app.agent.create_processor().get_tracker(conversation_id)
```

Comment:
There is no need to use the `processor here`. We can directly use
```suggestion
            tracker = app.agent.tracker_store.retrieve(conversation_id)
```

Reply:
As requested, we've implemented the method `exists` at the `TrackerStore`  interface, so this change ended up as follows:

```python
            if app.agent.tracker_store.exists(conversation_id):
```